numpy/matrixlib/tests/test_masked_matrix.py

```diff
@@ -79,10 +79,11 @@ def test_matrix_indexing(self):
     def test_pickling_subbaseclass(self):
         # Test pickling w/ a subclass of ndarray
         a = masked_array(np.matrix(list(range(10))), mask=[1, 0, 1, 0, 0] * 2)
-        a_pickled = pickle.loads(a.dumps())
-        assert_equal(a_pickled._mask, a._mask)
-        assert_equal(a_pickled, a)
-        assert_(isinstance(a_pickled._data, np.matrix))
+        for proto in range(2, pickle.HIGHEST_PROTOCOL + 1):
+            a_pickled = pickle.loads(a.dumps())
```

Comment:
this line is missing the `protocol` argument. I will first fix this in the PR about specifying pickle protocol as a kwarg in `np.ndarray.dumps` and `np.ndarray.dump`